src/compiler/typescript_to_rust/azle_generate/src/cdk_act/nodes/act_init_method_node.rs

```diff
@@ -0,0 +1,22 @@
+use proc_macro2::TokenStream;
+use quote::quote;
+
+use crate::cdk_act::{nodes, ToTokenStream, ToTokenStreams};
+
+pub struct ActInitMethodNode {
+    pub params: Vec<nodes::Param>,
+    pub body: TokenStream,
+}
+
+impl ToTokenStream for ActInitMethodNode {
+    fn to_token_stream(&self) -> TokenStream {
```

Comment:
pass cdk_name along.

Reply:
This would change the signature of `.to_token_stream`... I'm going to tackle this in another issue: https://github.com/demergent-labs/azle/issues/699